boards/hifive1b/include/periph_conf.h

```diff
@@ -28,7 +30,7 @@ extern "C" {
  * @{
  */
 /* As defined in boards/hifive1/board.c CPU_DESIRED_FREQ **/
```

Comment:
is the comment correct?

Reply:
Indeed, haven't checked the comment. Anyway, I have a branch where I rework the file organization more in depth, and this comment is gone there. It still needs a bit of work, but I'll PR it soon.

Reply:
I dropped this change anyway.